### PR TITLE
[close #207] cdc: modify ticdc dockerfile for tikv-cdc

### LINKED ISSUE #207: cdc: add tikv-cdc dockerfile
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

cdc: add tikv-cdc dockerfile 

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?
 modify ticdc dockerfile for tikv-cdc

Issue Number: close #207

Problem Description: **TBD**

### What is changed and how does it work?

 modify ticdc dockerfile for tikv-cdc

### CODE REVIEW

File: cdc/deployments/tikv-cdc/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM golang:1.18.0-alpine3.15 as builder
+RUN apk add --no-cache git make bash
+WORKDIR /go/src/github.com/tikv/migration/cdc
+COPY . .
+ENV CDC_ENABLE_VENDOR=0
+RUN make
```
Comment (haojinming) [inline cdc/deployments/tikv-cdc/docker/Dockerfile:6]: make release

File: cdc/deployments/tikv-cdc/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+FROM alpine:3.15
+RUN apk add --no-cache tzdata bash curl socat
+COPY --from=builder /go/src/github.com/tikv/migration/cdc/bin/tikv-cdc /usr/bin/
+# Tikv-cdc use ticdc operator to run tikv-cdc server, ticdc operator will use '/cdc' to start server
```
Comment (haojinming) [inline cdc/deployments/tikv-cdc/docker/Dockerfile:11]:
```suggestion
# TiKV-CDC use ticdc operator to run tikv-cdc server, ticdc operator will use '/cdc' to start server 
```

File: cdc/deployments/tikv-cdc/docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+WORKDIR /go/src/github.com/tikv/migration/cdc
+COPY . .
+ENV CDC_ENABLE_VENDOR=0
+RUN make
+
+FROM alpine:3.15
+RUN apk add --no-cache tzdata bash curl socat
+COPY --from=builder /go/src/github.com/tikv/migration/cdc/bin/tikv-cdc /usr/bin/
+# Tikv-cdc use ticdc operator to run tikv-cdc server, ticdc operator will use '/cdc' to start server 
+COPY --from=builder /go/src/github.com/tikv/migration/cdc/bin/tikv-cdc /cdc
```
Comment (haojinming) [inline cdc/deployments/tikv-cdc/docker/Dockerfile:12]: I think this is a temporary method?

**Review by pingyu — APPROVED** [review]
LGTM~

Suggest to add a merge CI (CI after PR merged) to automatically push a latest image to `hub.pingcap.net` later, for ease of future use.

**Review by haojinming — APPROVED** [review]
LGTM

**Pull request merged**